sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOIT.java

```diff
@@ -140,12 +137,12 @@ public interface JmsIOITOptions extends IOTestPipelineOptions, StreamingOptions
 
   @Parameterized.Parameters(name = "with client class {3}")
   public static Collection<Object[]> connectionFactories() {
-    return Collections.singletonList(
+    return ImmutableList.of(
         new Object[] {
           "vm://localhost", 5672, "jms.sendAcksAsync=false", ActiveMQConnectionFactory.class
         });
     // TODO(https://github.com/apache/beam/issues/26175) Test failure on direct runner due to
-    //  JmsIO read on amqp slow on Jenkins.
+    //  JmsIO read on amqp slow on CI (passed locally)
```

Comment:
turns out we still cannot un-ignore ampq case. It can pass consistently on my machine, but it still fails on CI